build/release/containers.sh

```diff
@@ -75,22 +75,21 @@ ENTRYPOINT ["/usr/bin/rookd"]
 EOF
     build_artifact $os $arch $tmpdir rook/rookd rookd
     rm -fr $tmpdir
-
-    # TODO: build the client and toolbox for arm
-    [[ ${arch} == "amd64" ]] || return 0
  
     echo "Building the rook-client container"
     mkdir $tmpdir
     cat <<EOF > $tmpdir/Dockerfile
-FROM ubuntu
-RUN apt-get update && \
-    apt-get install -yq --no-install-recommends wget s3cmd kmod module-init-tools sudo
```

Comment:
help me understand better how the `rook-client` pod would be able to load the `rbd` kernel module without some of these packages? (or their equivalents for Alpine that is)